numpy/lib/histograms.py

```diff
@@ -345,6 +345,15 @@ def _search_sorted_inclusive(a, v):
         a.searchsorted(v[-1:], 'right')
     ))
 
+def histogram_bin_edges(a, bins=10, range=None, weights=None):
+    """
+    Function to calculate the edges of the bins used by the histogram function. 
+    This function should be used when the edges of the bins are required 
+    without the overhead of calculating the histogram.
+    """
```

Comment:
This is a good start. You should take a look at [HOWTO_DOCUMENT](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections) - in particular, a parameters section and a couple examples would go a long way

Reply:
Done.